README.md

```diff
@@ -2,77 +2,124 @@
 
 # DDEV-CRON <!-- omit in toc -->
 
-- [Intro](#intro)
+- [Introduction](#introduction)
 - [Getting started](#getting-started)
 - [Implementation](#implementation)
+  - [./ddev/web-build/cron.conf](#ddevweb-buildcronconf)
+  - [\*.cron](#cron)
+- [Useful sites and debugging](#useful-sites-and-debugging)
 - [Examples](#examples)
+  - [Logging current time](#logging-current-time)
+  - [TYPO3 scheduler](#typo3-scheduler)
+  - [Drupal cron](#drupal-cron)
+  - [Laravel cron](#laravel-cron)
 
-## Intro
+## Introduction
 
 This DDEV add-on helps to execute a command in the web container based on a cron schedule. Cron is a classic Linux/Unix service with a well-known configuration syntax.
 
-The add-on
+The add-on:
 
 - Installs and runs the cron service inside the web container
-- Adds a sample cron configuration that adds to a file every minute.
+- Adds an example job that writes out the current time.
 - Required DDEV v1.19.3 or higher.
```

Comment:
I think we can remove this now. And IIRC the install.yaml checks and gives output anyway.

```suggestion
```